osd/idp_has_connectivity_problems.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "configured identity provider ${IDP_PROVIDER} has problems connecting",
+    "description": "Please verify the status of your identity server to restore the cluster’s operation. Documentation: https://docs.openshift.com/dedicated/4/authentication/dedicated-understanding-authentication.html",
```

Comment:
A bit more verbose: "Please verify the status of your identity server and update the IdP ${IDP_PROVIDER} configuration to restore the cluster's operation. See documentation: ..."